data-loader/core/src/main/java/com/scalar/db/dataloader/core/util/DecimalUtil.java

```diff
@@ -0,0 +1,40 @@
+package com.scalar.db.dataloader.core.util;
+
+import java.text.DecimalFormat;
+import java.text.DecimalFormatSymbols;
+import java.util.Locale;
+
+/** Utils for decimal handling */
+public class DecimalUtil {
+
+  /**
+   * Convert a Double to a non-scientific formatted string
+   *
+   * @param doubleValue Double value
+   * @return formatted double as a string
+   */
+  public static String convertToNonScientific(Double doubleValue) {
+    return createFormatter().format(doubleValue);
```

Comment:
If this method is frequently called, I think the DecimalFormat should be reused.

```java
  DecimalFormat DECIMAL_FORMAT = createFormatter();

  ...

    public static String convertToNonScientific(Double doubleValue) {
      return DECIMAL_FORMAT.format(doubleValue);
    }
```

Reply:
@komamitsu san, I had added this change but had to revert back based on Suzuki-san's [comment](https://github.com/scalar-labs/scalardb/pull/2388#discussion_r1882276327).

Reply:
Ah, it's used by multiple threads, right. Understood. Using ThreadLocal is might be an option, though.